apps/user/components/tips/view.coffee

```diff
@@ -1,48 +1,57 @@
 Backbone = require 'backbone'
 Cookies = require 'cookies-js'
-
 analytics = require '../../../../lib/analytics.coffee'
-mediator = require '../../../../lib/mediator.coffee'
+Promise = require 'bluebird-q'
 
 module.exports = class TipView extends Backbone.View
+  events:
+    'click': 'onClick'
+    'click .js-close' : 'close'
 
-  events: 
-    'click': 'handleClick'
-    'click .js-close' : 'removeTip'
+  initialize: ({ @user }) -> #
 
-  removeTip: (e) ->
+  onClick: (e) ->
     e.preventDefault()
-    e.stopImmediatePropagation()
-    @setClosed()
-    @remove()
+    e.stopPropagation()
 
-  setClosed: ->
-    analytics.track.click "Block tip closed", id: @model.id
-    Cookies.set @model.id, true
+    @collection.remove @model
 
-  handleClick: (e) ->
-    e.preventDefault()
-    e.stopPropagation()
+    switch @model.get 'trigger'
+      when 'new:channel'
+        # Opens the new channel dialog
+        $dropdown = $('.js-triggerable-channel-create')
+        $dropdown.addClass hoverClass = 'DropdownHover--hover'
 
-    @setClosed()
+        # Closes it when the body is clicked
+        $('body').one 'click', (e) ->
```

Comment:
Clicking indeed 'hides' it when you mouse out of it; but that's the default behavior anyway. There's an issue to make this not the default behavior (ie: once you start interacting, prop it open). So that is the real fix here, IMO.